hkm/static/hkm/svg/Kuvia_en.svg

```diff
@@ -1,19 +1,81 @@
-        </g>
-    </g>
-</svg>
\ No newline at end of file
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This svg looks like it can be minified in order to get a smaller file size.

Reply:
None of the other svg's are minified, im skipping this one in order to avoid messing up the sizing.